### ISSUE TITLE
Mutations support

### ISSUE THREAD

**Issue by alexandru-calinoiu** [yoshuawuyts/nanographql#3]
Planning to add mutations support?

**Comment by yoshuawuyts** [comment]
Those should already be supported. Let me know if anything's missing!

**Comment by alexandru-calinoiu** [comment]
Can you put on an example?

I send:

```
    gql`
        mutation Dostuff($input: SomeInput!) {
            mutationName(mutationInput: $input) {
                field1,
                field2
            }
        }
    `;
```

The request that get's over to the server does not have an operationName.

**Comment by yoshuawuyts** [comment]
Oh, whoops - yeah that sounds like a bug then. PR welcome!

**Comment by alexandru-calinoiu** [comment]
Yeah the regex there only expects 'query'

**Comment by yoshuawuyts** [comment]
Closed by #4 :tada:
